index.js

```diff
@@ -141,21 +142,23 @@ var scssContextParser = (function () {
         context.value = match[4].trim();
         context.scope = match[5] || 'private';
       }
-      if (lineNumberFor !== undefined) {
-        context.line = {
-          start : lineNumberFor(startIndex) + 1,
-          end : lineNumberFor(endIndex) + 1
-        };
-      }
     } else {
-      var codeStart = ctxCode.indexOf('{');
-      if (codeStart > 0) {
+      startIndex = ctxCode.indexOf('{');
```

Comment:
Does this assumes the opening brace is placed on the same line as the selector ?
Some users might format their rule sets like:
```css
.foo
{
  color: red;
}
```